src/ui-components/QuickView.tsx

```
@@ -218,16 +218,21 @@
 						</div>
 					</div>
 				</div>
-				<div className={loginUserId && !isNaN(loginUserId) ? 'flex justify-between' : 'flex justify-start'}>
-					{loginUserId && !isNaN(loginUserId) && <FollowersAndFollowing userId={loginUserId} />}
+				<div className={userId && !isNaN(userId) ? 'flex justify-between' : 'flex justify-start'}>
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)